components/ActivityForm/index.js

```diff
@@ -55,6 +110,10 @@ export default function ActivityForm({ onSubmit, initialData, isEditMode }) {
       .replace(/(\.)\s+/g, "$1 ");
     newActivity.category = formData.getAll("category");
     newActivity.images = await uploadImages(images);
+    newActivity.lat = coordinates.lat;
+    newActivity.lng = coordinates.lng;
+    newActivity.city = selectedCity.value;
+    newActivity.country = selectedCountry.value;
```

Comment:
Is there a reason for overwriting the values coming from 'FormData'?

Reply:
right, for city is not needed, but for country is required, because label and value have different values, otherwise the submit takes the countryCode and not the countryName(the label)